contracts/chain-adapters/Arbitrum_Adapter.sol

```diff
@@ -64,10 +54,6 @@ contract Arbitrum_Adapter is AdapterInterface {
     // L2 Gas price bid for immediate L2 execution attempt (queryable via standard eth*gasPrice RPC)
     uint256 public immutable l2GasPrice = 5e9; // 5 gWei
 
-    // Gas limit for immediate L2 execution attempt (can be estimated via NodeInterface.estimateRetryableTicket).
-    // NodeInterface precompile interface exists at L2 address 0x00000000000000000000000000000000000000C8
-    uint32 public immutable l2GasLimit = 2_000_000;
```

Comment:
Gas limit for sending tokens should be a lot lower than for sending messages

Reply:
nit: any reason not to make these both constants rather than hardcoding them?

Reply:
you mean instead of `uint256 public immutable` do `uint256 public constant`? Is that a gas savings?

Reply:
Sorry, yeah, that too (constant is better than immutable!), but I just meant giving them names, which you did!